src/components/common/Label/styled.js

```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components';
+
+export const StyledLabel = styled.label`
+  font-weight: bold;
```

Comment:
I'd suggest creating a theme file which contains variables for various situations. A bold variable would be good. If for example you built out a massive website then you decided you want the font-weight to change from bold to bolder or whatever it's one change rather than 20 for example. Hopefully that makes sense.